nipoppy/data/examples/sample_bids_dataset_description.json

```
@@ -0,0 +1,13 @@
+{
+    "Name": "TODO: CHANGEME",
```

Can disclose and promote glorious nipoppy via prefix:


```suggestion
    "Name": "NiPoppy Project: TODO CHANGEME",
```